spec-tests/src/test/java/org/candlepin/spec/imports/ImportSuccessSpecTest.java

```diff
@@ -0,0 +1,357 @@
+ *
+ * Red Hat trademarks are not licensed under GPLv2. No permission is
+ * granted to use or replicate Red Hat trademarks that are incorporated
+ * in this software or its documentation.
+ */
+
+package org.candlepin.spec.imports;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import org.candlepin.dto.api.client.v1.AttributeDTO;
+import org.candlepin.dto.api.client.v1.BrandingDTO;
+import org.candlepin.dto.api.client.v1.CdnDTO;
+import org.candlepin.dto.api.client.v1.ConsumerDTO;
+import org.candlepin.dto.api.client.v1.ContentDTO;
+import org.candlepin.dto.api.client.v1.ImportRecordDTO;
+import org.candlepin.dto.api.client.v1.ImportUpstreamConsumerDTO;
+import org.candlepin.dto.api.client.v1.OwnerDTO;
+import org.candlepin.dto.api.client.v1.PoolDTO;
+import org.candlepin.dto.api.client.v1.ProductDTO;
+import org.candlepin.dto.api.client.v1.ProvidedProductDTO;
+import org.candlepin.dto.api.client.v1.SubscriptionDTO;
+import org.candlepin.dto.api.client.v1.UpstreamConsumerDTO;
+import org.candlepin.dto.api.client.v1.UserDTO;
+import org.candlepin.spec.bootstrap.assertions.OnlyInStandalone;
+import org.candlepin.spec.bootstrap.client.ApiClient;
+import org.candlepin.spec.bootstrap.client.ApiClients;
+import org.candlepin.spec.bootstrap.client.SpecTest;
+import org.candlepin.spec.bootstrap.data.builder.Branding;
+import org.candlepin.spec.bootstrap.data.builder.Consumers;
+import org.candlepin.spec.bootstrap.data.builder.Content;
+import org.candlepin.spec.bootstrap.data.builder.Export;
+import org.candlepin.spec.bootstrap.data.builder.ExportGenerator;
+import org.candlepin.spec.bootstrap.data.builder.Owners;
+import org.candlepin.spec.bootstrap.data.builder.ProductAttributes;
+import org.candlepin.spec.bootstrap.data.builder.Products;
+import org.candlepin.spec.bootstrap.data.util.Importer;
+import org.candlepin.spec.bootstrap.data.util.UserUtil;
+
+import com.fasterxml.jackson.databind.JsonNode;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+import java.util.List;
+import java.util.Map;
+import java.util.Set;
+import java.util.function.Consumer;
+import java.util.function.Function;
+import java.util.stream.Collectors;
+
+@SpecTest
+@OnlyInStandalone
+public class ImportSuccessSpecTest {
+
+    private static final String UNMAPPED_ATTRIBUTE_NAME = "unmapped_guests_only";
+    private static final String EXPECTED_CONTENT_URL = "/path/to/arch/specific/content";
+
+    private ApiClient admin;
+    private OwnerDTO owner;
+    private ApiClient userClient;
+    private ConsumerDTO consumer;
+    private Importer importer;
+
+    @BeforeEach
+    public void beforeAll() {
+        admin = ApiClients.admin();
+        owner = admin.owners().createOwner(Owners.random());
+        UserDTO user = UserUtil.createUser(admin, owner);
+        userClient = ApiClients.basic(user);
+        consumer = userClient.consumers().createConsumer(Consumers.random(owner));
```

Comment:
Should this be in the before method? I see this consumer is only used in 1 test method

Reply:
Done.